packages/stark-ui/src/modules/session-ui/entities/stark-session-ui-config.ts

```diff
@@ -0,0 +1,19 @@
+import { InjectionToken } from "@angular/core";
+
+/**
+ * The InjectionToken version of the config name
+ */
+export const STARK_SESSION_UI_CONFIG: InjectionToken<StarkSessionUiConfig> = new InjectionToken<StarkSessionUiConfig>(
+	"StarkSessionUiConfig"
+);
+
+/**
+ * Definition of the configuration object for the Stark Session service
+ */
+export class StarkSessionUiConfig {
+	/**
+	 * Router state for the Login page where the user can choose a profile and use it to impersonate himself as someone else.
+	 * This state is only used in DEVELOPMENT.
+	 */
+	public enableTimeoutWarningDialogEffects?: boolean;
```

Comment:
As we discussed, can you rename this option to "timeoutWarningDialogDisabled" so we keep consistency with the way the options in `stark-app-config` are named?